app/models/trip.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'trip_dashboard'
+require 'time'
+
+class Trip < ActiveRecord::Base
+  belongs_to  :condition
+  belongs_to  :station
+  belongs_to  :subscription_type
+
+  extend TripDashboard
+
+  def end_station
+    Station.find(end_station_id)
```

Comment:
this looks like it could potentially be an association method rather than a custom method